packages/framework/tests/Unit/Model/AdvancedSearch/AdvancedSearchQueryBuilderExtenderTest.php

```diff
@@ -19,6 +19,7 @@ public function testExtendByAdvancedSearchData()
         $ruleData->operator = 'testOperator';
         $ruleData->value = 'testValue';
 
+        /** @var \Shopsys\FrameworkBundle\Model\AdvancedSearch\AdvancedSearchRuleData[]|null[] $advancedSearchData */
```

Comment:
This cannot be array of nulls, right?

Reply:
Not array of nulls but there might be a null value.

https://github.com/shopsys/shopsys/blob/762a90cc3543b33bce1daca2feb8c48adcd3e79a/packages/framework/tests/Unit/Model/AdvancedSearch/AdvancedSearchQueryBuilderExtenderTest.php#L23

Reply:
```suggestion
        /** @var array $advancedSearchData */
```